www/_posts/2020-07-18-uiwebview-warning.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+author:
+    name: Niklas Merz
+    url: https://twitter.com/niklasmaerz
+title:  "UPDATED: How to handle the 'Deprecated API Usage - UIWebView' warning while uploading to the App Store"
+categories: howto
+tags: ios
+---
+
+We recently posted [instructions](/howto/2020/03/18/wkwebviewonly.html) how to update your apps to remove all `UIWebView` references, because Apple now rejects all apps using `UIWebView`.
+
+We are talking about this warning:
+
+> ITMS-90809: Deprecated API Usage - Apple will stop accepting submissions of apps that use UIWebView APIs.
+
+Some things have changed and new versions of cordova-ios have been released since the last post.
+
+<!--more-->
+
+## Update cordova-ios to version 6.0.0 or newer
+
+Please update to the latest cordova-ios version to get the best compatibility with recent iOS changes. As of this writing the current version is 6.1.0. Version 6.0.0 of cordova-ios moved `WKWebView` support into cordova-iOS and removed `UIWebView` code. Due to this change, the `cordova-plugin-wkwebview-engine` plugin is obsolete and will not work with this release. If you have this plugin installed, it is safe to remove with `cordova plugin remove cordova-plugin-wkwebview-engine`.
+
+Additionaly, `WKURLSchemeHandler` support has been introduced with this release. Using a custom scheme to serve your app content through fixes CORS issues that exist because of the strict security policies that `WKWebView` has applied to the `file` scheme. You can easily configure your Cordova project to use a custom scheme by setting the preference options `scheme` and `hostname` in the `config.xml` file.
```

Comment:
We may also want to note that changing schemes will cause clients to lose access to their current web storage, such as local storage, indexed DB, etc.

It's possible to migrate this data but Cordova doesn't currently handle this.

Reply:
Yeah a good idea to warn about that. Just pushed one sentence.